tests/core/a-mixin.test.js

```diff
@@ -216,6 +217,44 @@ suite('a-mixin', function () {
       });
     });
   });
+
+  suite('parseComponentAttrValue', function () {
```

Comment:
Moved and adapted from `component.test.js` where it tested the `parseAttrValue` method. Since this PR causes it to only be used by `a-mixin`, the tests were moved as well.